### ISSUE TITLE
Accessibility: unlabeled buttons when using talkback

### ISSUE THREAD

**Issue by shortwavesurfer2009** [cake-tech/cake_wallet#2362]
**Describe the bug**

These buttons are unlabeled when using the TalkBack screen reader.

**To Reproduce**
1. Enable the TalkBack screen reader in Android System Settings under Accessibility.
2. Open monero.com wallet
3. Tap the top bar where it says connecting, attempting to sync, or synchronized.
4. Double tap to activate.
5. Cap Manage Nodes, then Double Tap to activate.

**Expected behavior**

The buttons should be properly labeled.

**Screenshots**

<img width="1080" height="2400" alt="Image" src="https://github.com/user-attachments/assets/db6afb20-f7f5-422a-a846-d4fc845204ac" />
<img width="1080" height="2400" alt="Image" src="https://github.com/user-attachments/assets/3c7bcdd0-f406-4671-bc46-5de253601558" />
<img width="777" height="1727" alt="Image" src="https://github.com/user-attachments/assets/f640990f-a29a-406c-b9a2-f9ae7139e3f4" />

**Platform:**
 - OS: [e.g. iOS 15.1, Android 14] Android 15
 - Device: [e.g. iPhone 14, Galaxy S21] OnePlus Nord N200
 - Cake Wallet Version: [e.g. 4.12.1] Monero.com v5.1.0

**Additional context**

**Comment by shortwavesurfer2009** [comment]
> Hi [@shortwavesurfer2009](https://github.com/shortwavesurfer2009) ,
> 
> Thanks for reaching out!
> 
> For quicker assistance, please contact our support team at [sdeskcental@gmail.com](mailto:sdeskcental@gmail.com) — they’re available and ready to help you promptl

SCAM
